.circleci/config.yml

```diff
@@ -1,66 +1,107 @@
-version: 2
-
-geth-defaults: &geth-defaults
-  docker:
-    - image: circleci/golang:1.12
-
-rust-defaults: &rust-defaults
-  docker:
-    - image: circleci/rust:1.37.0
-
-end-to-end-defaults: &end-to-end-defaults
-  docker:
-    - image: celohq/node10-gcloud
-  environment:
-    CELO_MONOREPO_BRANCH_TO_TEST: master
-
-jobs:
-  unit-tests:
-    <<: *geth-defaults
-    working_directory: ~/repos/geth
-    steps:
-      - attach_workspace:
-          at: ~/repos
-      - run: build/env.sh go run build/ci.go test
-
-  coverage:
-    <<: *geth-defaults
-    working_directory: ~/repos/geth
-    steps:
-      - attach_workspace:
-          at: ~/repos
-      - run: build/env.sh go run build/ci.go test -coverage
-      - run: bash <(curl -s https://codecov.io/bash)
-
-  lint:
-    <<: *geth-defaults
+version: 2.1
+executors:
+  golang:
+    docker:
+      - image: circleci/golang:1.13
     working_directory: ~/repos/geth
-    steps:
-      - checkout
-      - run: build/env.sh go run build/ci.go lint
-
-  bls-zexe:
-    <<: *rust-defaults
+  e2e:
+    docker:
+      - image: celohq/node10-gcloud
+    working_directory: ~/repos/celo-monorepo/packages/celotool
+    environment:
+      GO_VERSION: "1.13.7"
+      CELO_MONOREPO_BRANCH_TO_TEST: mc/geth1.9-final
+jobs:
+  build-rust:
```

Comment:
Maybe `build-bls-zexe` is a better name? Otherwise it makes you think this job builds rust 😉